Sources/PackageDescription/PackageDescriptionSerialization.swift

```diff
@@ -407,6 +463,16 @@ extension Target.PluginUsage: Encodable {
 }
 
 extension SwiftVersion: Encodable {
+    /// Encodes this value into the given encoder.
+    ///
+    /// If the value fails to encode anything, `encoder` will encode an empty
+    /// keyed container in its place.
+    ///
+    /// This function throws an error if any values are invalid for the given
+    /// encoder's format.
+    ///
+    /// - Parameters:
+    ///   - encoder: The encoder to write data to.
```

Comment:
I don't think this description is accurate, either.

Reply:
Same comment as above about this source file.